brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/OrderPreviewPresenter.swift

```diff
@@ -57,7 +57,7 @@ final class OrderPreviewPresenter: NSObject, Presenter, OrderPreviewActionRespon
             .termsAndConditions: [
                 LabelViewModel.text("By placing this order you agree to our Terms and Conditions")
             ],
-            .confirm: [
+            .submit: [
```

Comment:
just curious.. why the rename? 🤔

Reply:
just to keep it unified with the billing details screen. will make it reusable.